README.md

```diff
@@ -220,6 +223,53 @@ The intended general flow for the natural language processing is as follows:
 1. The emit target listener within the command script gets control.  It should validate that it has the needed parameter values and then proceed with the same command processing that occurs if it had been received via the regex path.
 
 
+## Commands
+
+- `nlc status` - Show current status of the Hubot NLC classifier.
+- `nlc list|show` - List all the classifiers in the NLC instance.
+- `nlc train|retrain` - Retrain the Hubot NLC classifier.
+- `nlc auto approve [on|off|true|false]` - Toggle auto approve setting for training NLC statements.
+
```

Comment:
1. Use `Watson NLC` instead of `Hubot NLC`.  I see this in a couple places.

1. The C in NLC is for classifier, so technically we are duplicating the word.  I know what you mean because I struggle with this one every single time.  

Here's some suggestions:
> Show the status of the Watson NLC instance currently being used by the bot. 

> List all the Watson NLC instances. 


> Train a new Watson NLC instance with the current training data.


It's not possible to retrain an NLC instance, that's why I'd remove that word.

> Toggle auto approve of new NLC statements learned from usage.

Also, copy (or point to) warning about using auto-approve from https://github.com/ibm-cloud-solutions/hubot-ibmcloud-cognitive-lib/blob/master/README.md#environment-variables